model/task_queue_service.go

```diff
@@ -163,14 +164,28 @@ func (t *basicCachedDispatcherImpl) Refresh() error {
 	grip.Debug(message.Fields{
 		"ticket":       "EVG-6289",
 		"function":     "Refresh",
+		"message":      "refresh was successful",
 		"order_length": len(t.order),
 		"units_length": len(t.units),
 	})
 
 	return nil
 }
 
-func shouldRefreshCached(ttl time.Duration, lastUpdated time.Time) bool {
+func shouldRefreshCached(ttl time.Duration, lastUpdated time.Time, distroID string) bool {
+	if time.Since(lastUpdated) > ttl {
+		grip.Debug(message.Fields{
```

Comment:
An FYI for next time, but no need to fix since this is temporary logging: There's a nit: `grip.DebugWhen`, which you can use instead of `if`.

Reply:
Fixed so I remember this next time.